src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMRanker.scala

```diff
@@ -147,20 +165,26 @@ class LightGBMRankerModel(override val uid: String, override val model: LightGBM
 object LightGBMRankerModel extends ConstructorReadable[LightGBMRankerModel] {
   def loadNativeModelFromFile(filename: String, labelColName: String = "label",
                               featuresColName: String = "features",
-                              predictionColName: String = "prediction"): LightGBMRankerModel = {
```

Comment:
likewise alot of clutter here

Reply:
done, changed models to use ComplexParamsWritable based on discussion